roles/buildbot-worker/tasks/main.yml

```diff
@@ -12,6 +12,7 @@
       # repoclient needs these:
       - python3-requests
       - python3-gi
+      - subversion
```

Comment:
I'd move this and the one below above the `# repoclient needs these:` comment since it's a general build dependency.

Reply:
Ah, good catch, thanks.